src/vehicle.cpp

```diff
@@ -3756,9 +3766,49 @@ void vehicle::idle(bool on_map) {
 
     if( on_map ) {
         update_time();
+        if(scoop_on){
+            operate_scoop();
+        }
+    }
+}
+void vehicle::operate_scoop()
+{
+    std::vector<int> scoops = all_parts_with_feature( "SCOOP" );
+    auto veh_points = get_points();
+    for( int scoop : scoops ) {
+        int chance_to_damage_item = parts[scoop].info().dmg_mod;
```

Comment:
`dmg_mod` is supposed to affect damage from collision positively. Here you use it to mean something completely different and in fact opposite - to lower the chance of damage. It would be better to either invert the value (`chance = 100 - dmg_mod`) or just hardcode the damage chance.